typescript/simple-mercari-web/src/components/Listing/Listing.tsx

```diff
@@ -9,16 +9,23 @@ export const Listing: React.FC<{}> = () => {
     image: "",
   };
   const [values, setValues] = useState(initialState);
```

Comment:
`useState<File>` と同じように、<> の中に型を定義できます。

```suggestion
  const [values, setValues] = useState<formData>(initialState);
```

`formData`（名前は自由につけてください）は、下のように別で定義します
```
interface formData {
  name: string
  category: string
  image: string
}
```